libraries/Microsoft.Bot.Builder/CloudAdapterBase.cs

```diff
@@ -278,11 +278,6 @@ protected async Task<InvokeResponse> ProcessActivityAsync(AuthenticateRequestRes
         /// <returns>A <see cref="ClaimsIdentity"/> with the audience and appId claims set to the appId.</returns>
         protected ClaimsIdentity CreateClaimsIdentity(string botAppId)
         {
```

Comment:
Hi @garypretty, can we mimic the logic we have in  [botframeworkadapter](https://github.com/Microsoft/botbuilder-dotnet/blob/589a0026af8a312b0bba8a9346a347ada2642303/libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs#L261) rather than just removing the validation?
We discussed with the rest of the team and we think is better to fallback the AppId to string.Empty (in case is null)

Reply:
Done.